src/SDKs/Azure.Base/data-plane/Azure.Base/Azure.Base.props

```diff
@@ -1,11 +1,12 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Project references -->
-  <ItemGroup Condition="'$(UseProjectReferenceToAzureBase)'=='true'">
```

Comment:
Did you mean to include this file in the PR?

Reply:
Yes, I talked to  @weshaggard and @AlexGhiondea and we agreed to switch to P2P references till we ship.

Reply:
Could you instead just set this property to true above instead of removing the condition? That way we can simply delete the one line later to revert this.

Reply:
@AlexGhiondea suggested to just remove the condition, but I will change it as you said.